test/webhook_test.rb

```diff
@@ -2,19 +2,19 @@
 
 class WebhookTest < Test::Unit::TestCase
   test 'create should create a webhook' do
-    fake 'webhooks', :method => :post, :status => 201, :body => load_fixture('webhook')
```

Comment:
It's strange the the auto-correct added parens in some in other places but not here.